test/local/senders/email.js

```diff
@@ -443,8 +466,8 @@ describe(
             'test utm params for ' + type,
             function () {
               var syncLink = mailer._generateUTMLink(config.get('smtp').syncUrl, {}, type, 'connect-device')
-              var androidLink = mailer._generateUTMLink(config.get('smtp').androidUrl, {}, type, 'connect-android')
-              var iosLink = mailer._generateUTMLink(config.get('smtp').iosUrl, {}, type, 'connect-ios')
+              var androidLink = config.get('smtp').androidUrl
```

Comment:
Ugh, this might be been check in by accident while debugging email errors, reverting.